test/test_parser.py

```diff
@@ -0,0 +1,44 @@
+import os
+
+from pyfakefs import fake_filesystem_unittest
+
+import cabinets
+from cabinets import Parser
+
+
+class MockTextParser(Parser):
+    @classmethod
+    def load_content(cls, content: bytes, **kwargs):
+        return {'mock-parser': content.decode('utf-8')}
+
+
+class TestParserArgument(fake_filesystem_unittest.TestCase):
+    fixture_path = os.path.join(os.path.dirname(__file__), 'fixtures')
+
+    def setUp(self):
+        self.setUpPyfakefs()
+        self.fs.add_real_directory(self.fixture_path)
+
+    def test_read_plain_text_standard_parser(self):
+        protocol = 'file'
+        filename = os.path.join(self.fixture_path, 'sample_small.txt')
+        data = cabinets.read(f'{protocol}://{filename}', parser=True)
+        expected = "I am sample text!"
+        self.assertIsInstance(data, str)
```

Comment:
Doesn't really matter, but you could move the assertion up since you already have everything you need and it would fail even before trying to think about expected output.

```suggestion
        self.assertIsInstance(data, str)
        expected = "I am sample text!"
```

Reply:
Changed